setup

```diff
@@ -1,9 +1,7 @@
 #!/usr/bin/env python3
-
```

Comment:
[Following PEP8](https://www.python.org/dev/peps/pep-0008/#blank-lines) I'm against removing this line. It's a subjective interpretation, but I think the blank line separates the logic here (shebang from import section).